### PR TITLE
Add 24dp shadow (fixes #1606)

### LINKED ISSUE #1606: Feature request: styles for deeper shadows
Is this a new component, or a missing feature in an existing one?
A missing feature.

Component name:
Card (sort of -- shadows aren't a "component" but the MDL shadow styles are documented on the MDL Card documentation page).

Material Design spec URL for the component (if any):
https://www.google.com/design/spec/what-is-material/elevation-shadows.html

What does this component or feature allow you to do which isn’t possible at the moment?
Deeper shadows.  MDL provides `mdl-shadow--2dp` through `mdl-shadow--16dp`, which is great as far as it goes, but the MD spec calls for shadows between 1dp and 24dp.

Please provide any URLs or screenshots of good examples of usage of this component or feature that you may have come across.
The Material Design "Dialogs" page shows dialog boxes with "Dialog elevation: 24dp": https://www.google.com/design/spec/components/dialogs.html#dialogs-specs -- there's no dialog boxes in MDL yet (I know about #781) but if we had the shadow mixins we could make our own easily enough.

### PR DESCRIPTION
**Pull request by surma**
I used my highly trained designer eye to make the shadow look like the one [in the spec](https://www.google.com/design/spec/what-is-material/elevation-shadows.html#). Please check!

### CODE REVIEW

**Comment by addyosmani** [comment]
Where possible, it would be great to loop in @shyndman and the team as a sanity check here :)

**Comment by surma** [comment]
I’ll whip up a screenshot...

**Comment by surma** [comment]
screenshot 2015-09-28 16 24 30

File: src/_mixins.scss [hunk 1]
```diff
@@ -245,6 +245,12 @@
               0  8px 10px -5px rgba(0, 0, 0, $shadow-key-umbra-opacity);
 }
 
+@mixin shadow-24dp() {
+  box-shadow: 0 24px 36px 4px rgba(0, 0, 0, $shadow-key-penumbra-opacity),
```
Comment (shyndman) [inline src/_mixins.scss]: Here are the shadows we recommend:
```
box-shadow: 0px 9px 46px 8px $shadow-key-penumbra-opacity,
            0px 11px 15px -7px $shadow-ambient-shadow-opacity,
            0px 24px 38px 3px $shadow-key-umbra-opacity;
```

**Comment by surma** [comment]
screenshot 2015-09-28 17 54 00

Replaced my interpolated styles with the one from @shyndman (Thanks!).

**Comment by addyosmani** [comment]
LGTM for a merge. Appreciate the comments @shyndman and thanks for updating, @surma!

**Pull request closed without merging**